lua/conform/formatters/vsg.lua

```diff
@@ -0,0 +1,38 @@
+local function find_local_config()
+  local current_file = vim.api.nvim_buf_get_name(0)
+  local local_configs = { ".vsg.yaml", ".vsg.yml", ".vsg.json" }
```

Comment:
Since the official docs use `vsg_config.yaml` [in their examples](https://vhdl-style-guide.readthedocs.io/en/latest/editor_integration/vim.html#neovim), it probably makes sense to add support for that + variations as well.